pages/mesosphere/dcos/2.1/release-notes/2.1.0/index.md

```diff
@@ -28,50 +28,60 @@ This release provides new features and enhancements to improve the user experien
 
 ## Domain Sockets for Agent Executor Communication
 Agents and Executors now communicate over Unix Domain sockets making operators life easy in the presence of container overlay networks.
 
-### Marathon Fixed and Improved Issues
-
-https://github.com/mesosphere/marathon/blob/master/changelog.md
+# Marathon Fixed and Improved Issues
+For a detailed description on updates to Marathon, see the [changelog](https://github.com/mesosphere/marathon/blob/master/changelog.md).
 
-## Breaking changes
-Remove the octarine package from DC/OS. It was originally used as a proxy for the CLI but is not used for this purpose, anymore.
+# Breaking changes
+- Removed the octarine package from DC/OS. It was originally used as a proxy for the CLI but is not used for this purpose anymore.
+- DC/OS Net now waits until agents become active before adding DNS entries for tasks on the agent to prevent resolving to unreachable addresses. (DCOS_OSS-5463)
+- Removed the avro-cpp package from DC/OS. It was originally used as part of the metrics-collection framework, which now relies on a different infrastructure.
+- Removed the spartan package from DC/OS. Is was deprecated in 1.11 and replaced by dcos-net.
+- Removed the toybox package from DC/OS. Is was used only by Spartan.
+- Removed the dcos-history-service from DC/OS. (DCOS-58529)
 
-DC/OS Net will now wait until agents become active before adding DNS entries for tasks on the agent to prevent resolving to unreachable addresses (DCOS_OSS-5463)
+# Fixed and Improved Issues
+- Zookeeper log messages are now being forwarded to syslog. (COPS-6128)
+- Fixed a critical error in Metronome where existing jobs appear to be lost after upgrade. (COPS-6092)
+- (COPS-5951, COPS-5827)
 
-Remove the avro-cpp package from DC/OS. It was originally used as part of the metrics-collection framework which now relies on a different infrastructure.
 
-Remove the spartan package from DC/OS. Is was deprecated in 1.11 and replaced by dcos-net.
+- Fixed an issue where in some rare circumstances, after upgrading a cluster, users were no longer able to launch tasks that use the UCR containerizer. (D2IQ-64507, COPS-5868)
+- Fixed an issue where image pull in UCR was not working for nvcr.io (missing ‘service’/‘scope’ parameters). (COPS-5804)
+- Upgraded Java to version 8u232 to align with previous DC/OS releases. (DCOS-62548, COPS-5738)
+- Fixed an issue where after a DC/OS upgrade, the executor resources used by tasks on the agent were being incorrectly counted against quota. (COPS-5725)
+- DC/OS Admin Router now allows large packages of files, up to 32GB, to the Package Registry. (D2IQ-61233, COPS-5615)
 
-Remove the toybox package from DC/OS. Is was used only by Spartan.
 
-Remove the dcos-history-service from DC/OS. (DCOS-58529)
+COPS-5428
+COPS-5931
```

Comment:
- Fixed a problem where the Mesos master could crash in some situations if an agent was drained and then subsequently reactivated.  (COPS-5931, MESOS-10116)